### PR TITLE
Add two missing links in TOC.md

### PR DESCRIPTION
**Pull request by uglyengineer**
Co-Authored-By: Feng Liyuan <sunrunaway@users.noreply.github.com>

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: TOC.md [hunk 1]
```diff
@@ -72,6 +72,7 @@
   + [TiFlash 常用运维操作](/tiflash/maintain-tiflash.md)
   + [TiUP 常用运维操作](/maintain-tidb-using-tiup.md)
   + [Ansible 常用运维操作](/maintain-tidb-using-ansible.md)
+  + [时区支持](/configure-time-zone.md)
```
Comment (SunRunAway) [inline TOC.md:75]: 建议把【时区支持】放在【日常巡检】上面。

File: TOC.md [hunk 2]
```diff
@@ -92,6 +93,7 @@
   + [锁冲突与 TTL 超时](/troubleshoot-lock-conflicts.md)
   + [TiCDC 常见问题](/ticdc/troubleshoot-ticdc.md)
   + [TiFlash 常见问题](/tiflash/troubleshoot-tiflash.md)
+  + [TiDB 内存控制文档](/configure-memory-usage.md)
```
Comment (SunRunAway) [inline TOC.md:96]: 建议把 [TiDB 内存控制文档] 放在 [CPU 占用过多导致读写延迟增加] 下面

File: TOC.md [hunk 3]
```diff
@@ -87,6 +89,7 @@
   + [TiDB 集群问题导图](/tidb-troubleshooting-map.md)
   + [热点问题处理](/troubleshoot-hot-spot-issues.md)
   + [CPU 占用过多导致读写延迟增加](/troubleshoot-cpu-issues.md)
+  + [TiDB 内存控制文档](/configure-memory-usage.md)
```
Comment (kissmydb) [inline TOC.md:92]: 这篇文档从标题来看更像是一个性能优化文档，放在「性能优化」-「软件优化」下面会不会更好？

File: TOC.md [hunk 4]
```diff
@@ -67,11 +67,13 @@
     + 使用 BR 工具
       + [使用 BR 进行备份与恢复](/br/backup-and-restore-tool.md)
       + [BR 备份与恢复场景示例](/br/backup-and-restore-use-cases.md)
+  + [时区支持](/configure-time-zone.md)
   + [日常巡检](/daily-check.md)
   + [TiCDC 运维操作及任务管理](/ticdc/manage-ticdc.md)
   + [TiFlash 常用运维操作](/tiflash/maintain-tiflash.md)
   + [TiUP 常用运维操作](/maintain-tidb-using-tiup.md)
   + [Ansible 常用运维操作](/maintain-tidb-using-ansible.md)
+
```
Comment (SunRunAway) [inline TOC.md:76]:
```suggestion
```

File: TOC.md (hunk 3 above)
Comment (SunRunAway) [reply TOC.md:92]: 放在「性能优化」-「软件优化」-「配置」里？

Comment (SunRunAway) [inline TOC.md:92]:
```suggestion
  + [TiDB 内存调优](/configure-memory-usage.md)
```

File: TOC.md [hunk 5]
```diff
@@ -67,6 +67,7 @@
     + 使用 BR 工具
       + [使用 BR 进行备份与恢复](/br/backup-and-restore-tool.md)
       + [BR 备份与恢复场景示例](/br/backup-and-restore-use-cases.md)
+  + [时区支持](/configure-time-zone.md)
```
Comment (yikeke) [inline TOC.md:70]:
```suggestion
  + [修改时区](/configure-time-zone.md)
```
顺便把 `configure-time-zone.md` 文章的标题也改成 `修改时区`？

File: TOC.md (hunk 3 above)
Comment (yikeke) [reply TOC.md]: 放在「性能优化」-「软件优化」-「配置」里？
-----
我觉得可以

Comment (yikeke) [reply TOC.md]: @kissmydb PTAL again

Comment (kissmydb) [reply TOC.md]: LGTM

File: TOC.md [hunk 6]
```diff
@@ -87,6 +88,7 @@
   + [TiDB 集群问题导图](/tidb-troubleshooting-map.md)
   + [热点问题处理](/troubleshoot-hot-spot-issues.md)
   + [CPU 占用过多导致读写延迟增加](/troubleshoot-cpu-issues.md)
+  + [TiDB 内存调优](/configure-memory-usage.md)
```
Comment (yikeke) [inline TOC.md:91]: Please move this to 性能调优 - 软件调优 - 配置. @uglyengineer @SunRunAway

File: TOC.md [hunk 7]
```diff
@@ -468,6 +470,7 @@
 + 版本发布历史
   + [发布版本汇总](/releases/release-notes.md)
   + v4.0
+    - [4.0.2](/releases/release-4.0.2.md)
```
Comment (yikeke) [inline TOC.md:473]:
```suggestion
```
This should not be included in this PR? @uglyengineer

File: releases/release-notes.md [hunk 8]
```diff
@@ -10,6 +10,7 @@ TiDB 历史版本发布声明如下：
 
 ## 4.0
 
+- [4.0.2](/releases/release-4.0.2.md)
```
Comment (yikeke) [inline releases/release-notes.md:13]:
```suggestion
```
This should not be included in this PR? @uglyengineer

File: TOC.md (hunk 3 above)
Comment (uglyengineer) [reply TOC.md]: done

File: releases/release-notes.md (hunk 8 above)
Comment (uglyengineer) [reply releases/release-notes.md:13]: yes

File: TOC.md (hunk 7 above)
Comment (uglyengineer) [reply TOC.md:473]: yes

File: TOC.md (hunk 6 above)
Comment (uglyengineer) [reply TOC.md]: done

File: TOC.md (hunk 7 above)
Comment (yikeke) [reply TOC.md:474]: But you did not commit my suggestion yet. Hmmm I'll commit it then.
